packages/automerge-repo-react-hooks/src/useRemoteAwareness.ts

```diff
@@ -11,17 +11,17 @@ export const peerEvents = new EventEmitter()
 
 export interface UseRemoteAwarenessProps<T> {
   /** The handle to receive ephemeral state on */
-  handle: DocHandle<T>
-  /** Our user ID */
+  handle?: DocHandle<unknown>
+  /** Our user ID - Automerge BroadcastChannel sometimes sends us our own messages; optionally filters them */
```

Comment:
This comment feels out of place. Maybe it should be [here](https://github.com/automerge/automerge-repo/pull/436/files#diff-4bbc5a79e075b8ee48d9a79ca2b9b5565f07848707cd0f5ab266d7e1f81ff8e9R55) instead, but it's fairly clear what's going on there...
```suggestion
  handle?: DocHandle<unknown>
  /** Our user ID */
```